deploy/base/gradle-v0.1.yaml

```diff
@@ -0,0 +1,362 @@
+                    ;;
+                *)
+                    echo "Unsupported Gradle version: ${GRADLE_VERSION}" >&2
+                    exit 1
+                    ;;
+            esac
+        else
+            echo "Could not set Gradle version from gradle/wrapper/gradle-wrapper.properties" >&2
+            exit 1
+        fi
+
+        # XXX: Workaround for com.github.sherter.google-java-format not working on Java 17
+        if [ $(find -type f -name 'build.gradle*' -print0 | xargs -0 grep -c com.github.sherter.google-java-format) -gt 0 ]; then
+            echo "Found com.github.sherter.google-java-format plugin. Forcing Java version to 11" >&2
+            SUPPORTED_JAVA_VERSION="11"
+        fi
+
+        JAVA_VERSION="$(params.JAVA_VERSION)"
+
+        [ -n "${JAVA_VERSION}" ] || JAVA_VERSION="${SUPPORTED_JAVA_VERSION}"
+
+        export JAVA_HOME="/usr/lib/jvm/java-${JAVA_VERSION}-openjdk"
+        export PATH="${JAVA_HOME}/bin:${PATH}"
+        export PATH="${GRADLE_HOME}/bin:${PATH}"
+        export LANG=en_US.UTF-8
+        export LC_ALL=en_US.UTF-8
+
+        if [ -n "$(params.ENFORCE_VERSION)" ]; then
+            gradle-manipulator -l "${GRADLE_HOME}" $(params.GRADLE_MANIPULATOR_ARGS) -DversionOverride=$(params.ENFORCE_VERSION) generateAlignmentMetadata || exit 1
+        else
+            gradle-manipulator -l "${GRADLE_HOME}" $(params.GRADLE_MANIPULATOR_ARGS) generateAlignmentMetadata || exit 1
+        fi
+
+        gradle -DAProxDeployUrl=file:$(workspaces.source.path)/hacbs-jvm-deployment-repo $@ || exit 1
+
+        # fix-permissions-for-builder
+        chown 1001:1001 -R $(workspaces.source.path)
+    - name: deploy-and-check-for-contaminates
+      image: "registry.access.redhat.com/ubi8/ubi:8.5"
+      securityContext:
+        runAsUser: 0
+      script: |
+        tar -cvvzf $(workspaces.source.path)/hacbs-jvm-deployment-repo.tar.gz -C $(workspaces.source.path)/hacbs-jvm-deployment-repo .
+        curl --verbose --data-binary @$(workspaces.source.path)/hacbs-jvm-deployment-repo.tar.gz http://localhost:2000/deploy
+
+        curl --verbose --fail http://localhost:2000/deploy/result -o $(results.contaminants.path)
+        cat $(workspaces.maven-settings.path)/sidecar.log
+      resources:
+        requests:
+          memory: "128Mi"
+          cpu: "10m"
+        limits:
+          memory: "512Mi"
+          cpu: "300m"
+    - name: analyse-dependencies
+      securityContext:
+        runAsUser: 0
+
+      image: quay.io/redhat-appstudio/hacbs-jvm-dependency-analyser:3dcd8603061c7ab2cedfe128c80cac9726721a74
+      args:
+        - path
+        - $(workspaces.source.path)/$(params.CONTEXT_DIR)/$(params.DEPENDENCY_ANALYSER_PATH)
+        - -s
+        - $(workspaces.source.path)/$(params.CONTEXT_DIR)/$(params.DEPENDENCY_ANALYSER_PATH)/java-sbom.json
+        - -c
+        - "$(results.dependencies.path)"
+        - -u
+        - "$(results.untrusted-dependencies.path)"
```

Comment:
This is old and shouldn't be here. But, it used to kick off the build of dependencies, but broke within the last few days.

Reply:
```
[process-build-requests] 2022-07-18 21:39:11,959 INFO  [io.quarkus] (main) hacbs-build-request-processor 1.0-SNAPSHOT on JVM (powered by Quarkus 2.7.5.Final) started in 0.558s.
[process-build-requests] 2022-07-18 21:39:11,960 INFO  [io.quarkus] (main) Profile prod activated.
[process-build-requests] 2022-07-18 21:39:11,960 INFO  [io.quarkus] (main) Installed features: [cdi, jgit, jsch, picocli, rest-client-reactive, smallrye-context-propagation, vertx]
[process-build-requests] Missing required option: '--gav=<gav>'
[process-build-requests] Usage: <main class> [--build-info=<buildInfo>] [--context=<context>]
[process-build-requests]                     --gav=<gav> [--message=<message>] [--scm-tag=<scmTag>]
[process-build-requests]                     [--scm-type=<scmType>] [--scm-url=<scmUrl>]
[process-build-requests]                     --recipes=<recipeRepos>[,<recipeRepos>...]
[process-build-requests]                     [--recipes=<recipeRepos>[,<recipeRepos>...]]...
[process-build-requests]       --build-info=<buildInfo>
[process-build-requests]
[process-build-requests]       --context=<context>
[process-build-requests]       --gav=<gav>
[process-build-requests]       --message=<message>
[process-build-requests]       --recipes=<recipeRepos>[,<recipeRepos>...]
[process-build-requests]
[process-build-requests]       --scm-tag=<scmTag>
[process-build-requests]       --scm-type=<scmType>
[process-build-requests]       --scm-url=<scmUrl>
```